src/harvester/harvester.service.ts

```diff
@@ -200,6 +200,13 @@ export class HarvesterService {
       await this.preClear(args.repository, lastProcessedBlock, deployment);
     }
 
+    // when this argument is true, we'll need to send a request to the RPC to get the transaction object for each event
```

Comment:
Wouldn't you say agree that this kind of knowledge is better communicated in the configuration/env? Is it possible to set this preference there vs a commented code in a service

Reply:
agreed, updated!